docker-compose-multi-persona.yaml

```diff
@@ -0,0 +1,202 @@
+version: '3'
+
+services:
+  postgres-inteli-api-alice:
+    image: postgres:14.1-alpine
+    container_name: postgres-inteli-api-alice
+    ports:
+      - 5432:5432
+    volumes:
+      - inteli-api-storage-alice:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=inteli
+  
+  postgres-identity-service-alice:
+    image: postgres:14.1-alpine
+    container_name: postgres-identity-alice
+    ports:
+      - 5433:5432
+    volumes:
+      - identity-storage-alice:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=dscp
+
+  dscp-identity-service-alice:
+    image: digicatapult/dscp-identity-service:latest
+    container_name: identity-service-alice
+    command: /bin/sh -c "
+      sleep 30 &&
+      npx knex migrate:latest &&
+      node app/index.js"
+    ports:
+      - 3002:3002
+    depends_on:
+      - dscp-node-alice
+      - postgres-identity-service-alice
+    environment:
+      - PORT=3002
+      - API_HOST=dscp-node-alice
+      - DB_HOST=postgres-identity-service-alice
+      - DB_PORT=5432
+      - DB_NAME=dscp
+      - DB_USERNAME=postgres
+      - DB_PASSWORD=postgres
+      - SELF_ADDRESS=5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
```

Comment:
isn't this Bob's address? Alice is `5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY`

Reply:
does it matter as long as the self addresses are set to actual members?